README.md

```diff
@@ -25,14 +25,22 @@ To get a local copy up and running follow these simple example steps.
 
 
 
-## Authors: Joseph Kwesi Kwentsir
+## Authors 
 
 👤 **Author1**
+Joseph Kwesi Kwentsir
 
 - GitHub: [@githubhandle](https://github.com/kwentsir)
 - Twitter: [@twitterhandle](https://twitter.com/jkwentsir)
 - LinkedIn: [LinkedIn](https://linkedin.com/in/josephkwentsir)
```

Comment:
## [OPTIONAL]

- [ ] Kindly add a screenshot of your project in your Readme file to make it more professional.